lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/core/CameraPosition.kt

```diff
@@ -1,11 +1,25 @@
 package dev.sargunv.maplibrecompose.core
 
 import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.Stable
 import androidx.compose.ui.unit.dp
 import io.github.dellisd.spatialk.geojson.Position
 
-@Immutable
+/**
+ * Defines how the camera is oriented towards the map.
+ *
+ * @param bearing Direction that the camera is pointing in, in degrees clockwise from north.
+ * @param target Target position to align with the center of the screen, i.e. where the camera is
+ *   pointing at.
+ * @param tilt The camera angle, in degrees, from the nadir (directly down). A value in the range of
+ *   `[0 .. 60]`
+ * @param zoom Zoom level at target. A value in the range of `[0 .. 25.5]`
+ * @param padding By default, the camera points at the center of the screen, i.e. it also rotates
+ *   and tilts around that point. By specifying a padding, this center point can be altered, for
+ *   example if you want to display a bottom sheet above the lower part of the map, focussing on a
+ *   POI in the upper part of the map.
+ */
+@Stable
```

Comment:
I'd leave a comment on why this is Stable, not Immutable (PaddingValues).

In the future we might need to change `PaddingValues` below to `PaddingValues.Absolute`, and if we do we'll want to change this back to Immutable